bin/deploy-services

```diff
@@ -49,15 +49,16 @@ if [ ! "$LOCAL_DEV" = true ]; then
   fi
   envsubst < $file | kubectl apply -f - -n harmony
 
-  # create the work reaper
-  file="services/work-reaper/config/service-template.yaml"
+  # create the cron service
+  file="services/cron-service/config/service-template.yaml"
   if [ ! -f "$file" ]; then
-    echo "work reaper template was not found."
+    echo "cron service template was not found."
     exit 1
   fi
   envsubst < $file | kubectl apply -f - -n harmony
+fi
```

Comment:
This line should be removed otherwise you get syntax error because the `fi` won't match on line 68.

Reply:
Fixed